### PR TITLE
feat(TimePicker): add timepicker

### LINKED ISSUE #402: Create TimePicker ui-component

### PR DESCRIPTION
**Pull request by zouxuoz**
resolve #402

### CODE REVIEW

File: client/components/ui/forms/TimePicker/__tests__/TimePicker.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    const component = renderer.create(
+      <TimePicker onChange={onChange} />
+    );
+
+    expect(component.toJSON()).toMatchSnapshot();
+  });
+
+  test('renders with selected value', () => {
+    const component = renderer.create(
+      <TimePicker selected={moment({
```
Comment (rosendi) [inline client/components/ui/forms/TimePicker/__tests__/TimePicker.js]: I think an one line it's ok here :-)

File: client/components/ui/forms/TimePicker/TimePicker.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React, { PropTypes } from 'react';
+import moment from 'moment';
+import withBEM from 'containers/withBEM';
+import Select from 'components/ui/forms/Select';
+import './TimePicker.css';
+
+const TimePicker = ({ block, element, selected, onChange }) => {
+  const hoursOptions = [...Array(24).keys()].map((_, i) => ({ value: i, text: i }));
```
Comment (rosendi) [inline client/components/ui/forms/TimePicker/TimePicker.js]: May be make a helper?

**Comment by rosendi** [comment]
lgtm

**Comment by radiansz** [comment]
lgtm

**Pull request merged**